src/core/functions/internal_functions/file/InternalModuleFile.ts

```diff
@@ -141,12 +141,12 @@ export class InternalModuleFile extends InternalModule {
         };
     }
 
-    generate_folder(): (relative?: boolean) => string {
+    generate_folder(): (absolute?: boolean) => string {
         return (relative = false) => {
```

Comment:
We'll want to update this variable name as well, otherwise I get the following error `Template parsing error, aborting. absolute is not defined`
```suggestion
        return (absolue = false) => {
```

Reply:
Oh yes, that was a miss, thank you. I've force-pushed a correctly spelled version.

Reply:
Perfect, thanks for fixing that!